bindings/c/CMakeLists.txt

```diff
@@ -283,107 +283,96 @@ if(NOT WIN32)
+        --upgrade-path "${FDB_PREV2_RELEASE_VERSION}" "${FDB_PREV_RELEASE_VERSION}" "${FDB_CURRENT_VERSION}"
         --process-number 1
         )
 
-      add_test(NAME fdb_c_upgrade_multi_threaded_630api
+      add_scripted_fdb_test(NAME fdb_c_upgrade_multi_threaded_prev3_api
         COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
         --build-dir ${CMAKE_BINARY_DIR}
         --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
-        --upgrade-path "6.3.23" "7.0.0" "7.1.9" "7.3.0" "7.1.9"
+        --upgrade-path "${FDB_PREV3_RELEASE_VERSION}" "${FDB_PREV2_RELEASE_VERSION}" "${FDB_PREV_RELEASE_VERSION}" "${FDB_CURRENT_VERSION}" "${FDB_PREV_RELEASE_VERSION}"
         --process-number 3
         )
 
-      add_test(NAME fdb_c_upgrade_multi_threaded_700api
+      add_scripted_fdb_test(NAME fdb_c_upgrade_multi_threaded_prev2_api
         COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
         --build-dir ${CMAKE_BINARY_DIR}
         --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
-        --upgrade-path "7.0.0" "7.1.9" "7.3.0" "7.1.9"
+        --upgrade-path "${FDB_PREV2_RELEASE_VERSION}" "${FDB_PREV_RELEASE_VERSION}" "${FDB_CURRENT_VERSION}" "${FDB_PREV_RELEASE_VERSION}"
         --process-number 3
         )
 
-      add_test(NAME fdb_c_upgrade_multi_threaded_710api
+      add_scripted_fdb_test(NAME fdb_c_upgrade_multi_threaded_prev_api
         COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
         --build-dir ${CMAKE_BINARY_DIR}
         --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
-        --upgrade-path "7.1.9" "7.3.0" "7.1.9"
+        --upgrade-path "${FDB_PREV_RELEASE_VERSION}" "${FDB_CURRENT_VERSION}" "${FDB_PREV_RELEASE_VERSION}"
         --process-number 3
         )
 
-      add_test(NAME fdb_c_cluster_wiggle
+      add_scripted_fdb_test(NAME fdb_c_cluster_wiggle
         COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
         --build-dir ${CMAKE_BINARY_DIR}
         --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
-        --upgrade-path "7.3.0" "wiggle"
+        --upgrade-path "${FDB_CURRENT_VERSION}" "wiggle"
         --disable-log-dump
         --process-number 3
         --redundancy double
         )
 
-      add_test(NAME fdb_c_wiggle_and_upgrade_latest
+      add_scripted_fdb_test(NAME fdb_c_wiggle_and_upgrade_latest
         COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
         --build-dir ${CMAKE_BINARY_DIR}
         --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
-        --upgrade-path "7.1.9" "wiggle" "7.3.0"
-        --disable-log-dump
-        --process-number 3
-        --redundancy double
-        )
-
-      add_test(NAME fdb_c_wiggle_and_upgrade_63
```

Comment:
It was specific test for wiggle in 6.3. The version is getting old and the test is expensive, I think we can remove it.